deps/datReader/Exd/Structs.h

```diff
@@ -494,9 +494,11 @@ namespace Excel
     uint16_t Icon;
     uint16_t Action;
     uint8_t StackMax;
+    int8_t padding0[1];
```

Comment:
Are you sure about this? for one, in the exd this is an unsigned field, also you sure this isn't already the casttimeline?

Reply:
putting a byte here was the only way i could get the values to marry up nicely, otherwise CastTime never gets the correct value. 

But yeah, I could make it a uint.

Reply:
@SapphireMordred are you suggesting that it is CastTimeline that should be used instead of CastTime? In my quest if I didn't put the padding in then CastTime = 1 and CastTimeLine = 3. 
I checked a video online for the quest and the cast should take 3 seconds, that's why I am confused.

Reply:
What i am saying is that it should just be uint8_t Unknown; instead of int8_t padding ;)

Reply:
haha ok, fair enough. Changed :)